app/javascript/components/widgets/messages/render-message.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { GreySubText } from '../../typography';
+import { fontSizes, leftPanelWidth } from '../../../lib/theme';
+import sunMoon from '../../../../assets/images/sunmoon.png';
+
+export const RenderLoadingMessage = () => (
```

Comment:
I feel like we can ditch the `Render` prefix on these components -- I think the rendering is implied

Reply:
I think that's a good idea. @theroyalblah friendly reminder that it's a good idea to rename the `render-message.jsx` filename as well, and to squash those changes into the original commit.